pov.md

```diff
@@ -17,19 +17,18 @@ in the graph, by pulling it up to the root and dragging its relationships along
 with it. So the same graph from 6's perspective would look like:
 
 ```
-          6
-          |
-    7-----+-----2
-                |
-                0
-          1-----+-----3
-          |           |
-        4-+-5       8-+-9
+        6
+        |
+  7-----+-----2
+              |
+        1-----0-----3
+        |           |
+      4-+-5       8-+-9
 ```
 
 This lets us more simply describe the paths between two nodes. So for example
 the path from 6-9 (which in the first graph goes up to the root and then down to
 a different leaf node) can be seen to follow the path 6-2-0-3-9
 
-This exercise involves taking an input graph and re-orientating it one one of
-the nodes. 
+This exercise involves taking an input graph and re-orientating it from the point
+of view of one of the nodes.
```

Comment:
It wasn't 100% clear to me what the original author tried to type, but I think this captures the intent.